ms_game_chat/backend/backend_app/consumer/chat/chats.py

```diff
@@ -396,7 +396,7 @@ def getAvatar(self, chat_name):
             result = str(avatar_url) if avatar_url else None
         else:
             print("Invalid URL")
-            result = '../../backend' + str(avatar_url) if avatar_url else None # doesnt work if avatar is url
+            result = str(avatar_url) if avatar_url else None # doesnt work if avatar is url
```

Comment:
@mariekart42 as i provide always a valid URL you could remove this block, or point to a default avatar with this url: https://api.dicebear.com/8.x/identicon/svg?seed=default